index.html

```diff
@@ -0,0 +1,62 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Glory Mugizi Portfolio</title>
+    <link
+      href="https://fonts.googleapis.com/css?family=Poppins"
+      rel="stylesheet"
+    />
+    <link rel="stylesheet" href="styles.css" />
+  </head>
+  <body>
+    <header>
+      <nav>
+        <ul class="tool-bar">
+          <li class="my-logo">MY LOGO</li>
+          <li class="toggle-menu">
+            <img src="images/humbger.png" alt="" srcset="" />
+          </li>
+        </ul>
+      </nav>
+      <section class="headline">
+        <div class="container heading">
+          <h1>I’m Glory Mugizi Glad to see you!</h1>
```

Comment:
- [ ] Great job on using the `h1` tag correctly 💯 Kindly use the right `font-family` and the right text `color` to match the design in Figma 👍 (this applies to all the texts in the page}

## Your design:
![image](https://user-images.githubusercontent.com/50721479/205102555-e31ae8f7-52aa-481b-ae0f-3b6d559bd7c6.png)

## Figma design:
![image](https://user-images.githubusercontent.com/50721479/205102622-8fed51c5-78cd-4cc4-8cb6-373c0d611602.png)